### PR TITLE
kemas muhammad hanif andika

### PR DESCRIPTION
**Pull request by kemasdika**
- [x] login & test
- [x] register
- [x] show product
- [x] show product by id
- [x] create product & test
- [x] edit product & test
- [x] update product
- [x] delete product & test
- [x] api doc

env
SECRET_KEY=rubiku
DB_USERNAME_TEST=postgres
DB_PASSWORD_TEST=456123
DB_NAME_TEST=ecommerce_test
DB_HOST_TEST=127.0.0.1
DB_DIALECT_TEST=postgres

DB_USERNAME_DEVELOPMENT=postgres
DB_PASSWORD_DEVELOPMENT=456123
DB_NAME_DEVELOPMENT=ecommerce_development
DB_HOST_DEVELOPMENT=127.0.0.1
DB_DIALECT_DEVELOPMENT=postgres

### CODE REVIEW

File: test/product.test.js [hunk 1]
```diff
@@ -0,0 +1,943 @@
...
+			request(app)
+				.delete(`/products/${dummyProductId}`)
+				.set('access_token', 'wrong access token')
+				.end((err, res) => {
+					const { body, status } = res;
+					if (err) {
+						return done(err);
+					}
+					expect(status).toBe(401);
+					expect(body).toHaveProperty('message', 'Invalid Account Or Password');
```
Comment (mhmmd-ysf) [inline test/product.test.js]: Ini beda sama yang diterima server

**Review by mhmmd-ysf — COMMENTED** [review]
Dilanjut lagi ya

File: test/product.test.js (hunk 1 above)
Comment (kemasdika) [reply test/product.test.js]: sudah dibanerin kak

**Comment by mhmmd-ysf** [comment]
Overall bagus, kurang dikit aja tersting masih ada yang belum pass, tapi masih ok.
Good Job!

**Pull request closed without merging**